pkg/masking/config_mask.go

```diff
@@ -73,6 +74,19 @@ func maskConfigSecret(line string) string {
 	return line
 }
 
+func MaskPAT(line string) string {
+	regexPattern := `--` + conf.KeyDremioPatToken + ` .*`
```

Comment:
I'm sorry for this as this is my fault..but this is a devilish little pattern I've created and it has tripped you up.

- One of two args is accepted for the dremio-pat-token config value (-t and --dremio-pat-token).
- There is a loss of what flags are accepted once the flag is converted into this config key conf.KeyDremioPatToken
- so this code will not mask someone using the -t version of the --dremio-pat-token flag!!!!!

So to prevent further bugs we would need to link to that flag configuration and get the valid values from that (not even sure off the top of my head how to do this). 

My suggestion:

- Link to the flag so we keep configuration in sync if you can (yes i get my original implementation did not do this properly either)
- failing that..remove the -t flag (and update all doc references to it)
- document this ..honestly tenuous link between conf.KeyDremioPatToken and the real flag value..which are only in sync by convention and could someday fall out of sync

Reply:
So I did test `--dremio-pat-token` and `-t` they worked ok. What was not happening was that we weren't sending in both via the test. (See below)

Reply:
I actually tested `-t` and `--dremio-pat-token` both are masked ok from my testing against a Kubernetes cluster. However the test did not send in both values (see below for that)